shapes/typeIndex.shacl

```diff
@@ -0,0 +1,28 @@
+PREFIX sh: <http://www.w3.org/ns/shacl#>
+PREFIX rdfs: <http://www.w3.org/2000/01/rdf-schema#>
+PREFIX solid: <http://www.w3.org/ns/solid/terms#>
+PREFIX : <#>
+
+:TypeIndexShape a sh:Shape ;
+    sh:scopeClass solid:ListedDocument, solid:UnlistedDocument, solid:TypeIndex .
```

Comment:
Are you using an old version of the ShaCL specification? `sh:scopeClass` changed to `sh:targetClass` ([issue](https://www.w3.org/2014/data-shapes/track/issues/175)).

```suggestion
    sh:targetClass solid:ListedDocument, solid:UnlistedDocument, solid:TypeIndex .
```

Note that this is not a constraint, and does not parallel the ShEx shape. `sh:targetClass` indicates how to a ShaCL validator should select nodes to validate in a dataset. In this case, it will try to validate all `solid:ListedDocument` and `solid:UnlistedDocument` against the TypeIndexShape as well, which may or may not be what we want.

An alternative might be
```suggestion
    sh:targetClass solid:TypeIndex ;
    sh:class solid:ListedDocument .
```
which makes sure all `solid:TypeIndex` are also `solid:ListedDocument`.

Reply:
I thought this constraint makes sure that the record has one of the classes `solid:ListedDocument, solid:UnlistedDocument, solid:TypeIndex`. However that is also not correct. It should have `solid:ListedDocument, solid:TypeIndex` or `solid:UnlistedDocument, solid:TypeIndex`
In any case, it might be completely wrong.